server/src/tool.clj

```diff
@@ -238,3 +239,14 @@
      (require 'criterium.core)
      (criterium.core/quick-bench ~@body)
      (flush)))
+
+(defn prod-conn-str
+  []
+  (str/trim (:out (shell/sh "./scripts/prod_connection_string.sh"))))
+
+(defn prod-conn []
```

Comment:
This is dangerous, because it encourages people to leave connections open forever. How about this:

```
(defmacro with-prod-conn
  "Usage: (with-prod-conn [my-conn]
            (sql/select my-conn [\"select 1\"]))"
  [[conn-name] & body]
  `(let [cluster-id# (-> (clojure.java.io/resource "config/prod.edn")
                        slurp
                        clojure.edn/read-string
                        :database-cluster-id)
         rds-cluster-id->db-config# (requiring-resolve 'instant.aurora-config/rds-cluster-id->db-config)
         start-pool# (requiring-resolve 'instant.jdbc.aurora/start-pool)]
     (with-open [~conn-name (start-pool# 1 (rds-cluster-id->db-config# cluster-id#))]
       ~@body)))
```

And then you also don't need `prod-conn-str` above.

Reply:
Oo, I like it!